project_national/fact_sheets_reference.yml

```diff
@@ -0,0 +1,104 @@
+schema_version: '0.3'
+os_version: 3.5.1
+os_sha: 22e1db7be5
+buildstock_directory: ../ # Relative to this file or absolute
+project_directory: project_national # Relative to buildstock_directory
+output_directory: national_baseline
+# weather_files_url: https://data.nrel.gov/system/files/156/BuildStock_TMY3_FIPS.zip
+weather_files_path: /shared-projects/buildstock/weather/BuildStock_TMY3_FIPS.zip
+
+sampler:
+  type: precomputed
+  args:
+    sample_file: ../buildstock_10k_20230520.csv # relative to file
+
+baseline:
+  n_buildings_represented: 136569411 # American Community Survey 2019 5-year, B25001, does not include AK, HI, and territories
+  skip_sims: true
+
+eagle:
```

Comment:
We should probably suggest standard values here to use in all yamls (different sets for small, med, full)

Reply:
Maybe enlist help from @lixiliu for this

Reply:
It's gonna be complicated based on how many upgrades and their applicability , let's start an excel file on this next week.